### PR TITLE
Support separate node keypair creations/signing

### PR DESCRIPTION
**Pull request by sgnn7**
CC: @wallnerryan

### CODE REVIEW

**Comment by wallnerryan** [comment]
Thanks for the PR, will review and test this out soon.

**Comment by wallnerryan** [comment]
@sgnn7 sorry I havent got to this recently, have you continued using this portion and its working well?

**Comment by sgnn7** [comment]
@wallnerryan Very much so. We needed this functionality so we forked the repo with this patchset and the changes have been running fine in our clusters for the last 2 months with about 20 full re-deployments and about 40-50 AutoScalingGroup node count changes without a single problem so far.

File: generate_flocker_certs.sh [hunk 1]
```diff
@@ -270,11 +337,10 @@ generate_and_sign_cert "$output_dir/control-service" \
...
-                         "/CN=node-$(uuidgen)/OU=$cluster_uuid" \
-                         "$cluster_keypair_path"
-done
+# XXX: Array passed as a name, not a value
+generate_and_sign_node_certs $cluster_keypair_path \
+                             "nodes[@]" \
+                             $output_dir
 echo
 
 echo "Generating API keypair"
```
Comment (wallnerryan) [inline generate_flocker_certs.sh:346]: not sure I like that we always name this "plugin" but this was introduced elsewhere and we can fix it later.

**Comment by wallnerryan** [comment]
@sgnn7 just tested this, LGTM. Small nit was that we are naming the api user `plugin` always (https://github.com/ClusterHQ/flocker-openssl/pull/2/files#diff-51449c18de82d2f8ea5d90aa7a902f92R347) , its not a change in this PR so im going to merge anyway, can be a seperate update.

**Pull request merged**